packages/new-editor/src/components/editor/variables/detail/Metadata.tsx

```diff
@@ -0,0 +1,51 @@
+import { Fieldset, SimpleSelect } from '@axonivy/ui-components';
+import { treeNodeValueAttribute } from '../../../../types/config';
+import {
+  asEnumMetadata,
+  metadataOptions,
+  metadataTypeDaytime,
+  metadataTypeEnum,
+  variableMetadataAttribute,
+  type EnumMetadata,
+  type MetadataType,
+  type Variable,
+  type VariableUpdates
+} from '../../data/Variable';
+import { PossibleEnumValues } from './PossibleEnumValues';
+
+type MetadataFieldsetProps = {
+  variable: Variable;
+  onChange: (updates: VariableUpdates) => void;
+};
+
+export const Metadata = ({ variable, onChange }: MetadataFieldsetProps) => {
+  const enumMetadata = asEnumMetadata(variable.metadata);
+
+  return (
+    <>
+      <Fieldset label='Metadata'>
+        <SimpleSelect
+          value={variable.metadata.type}
+          items={metadataOptions}
+          emptyItem={true}
+          onValueChange={(value: MetadataType) => {
+            const newMetadata = { type: value };
+            const updates: VariableUpdates = [];
+            switch (value) {
+              case metadataTypeDaytime:
```

Comment:
I think I know why you did hide the value behind a variable. It is because then you can quickly rename the values right?

Understandable, but IMO this makes the readability of the code much worse. 
If I compare `case metadataTypeDaytime` to `case 'daytime'` I can ready the second one much quicker. 
And if the reason above is the only one, I'm not sure if it is enough, because we cannot simply change those values. They are in variable.yaml files in the project so, we need to convert customers projects if we want to change these... 
So IMO I would value the code readability much higher here than the possibility of refactoring the code a little bit faster. 
Or do you have another reason why you do not simply use the values?

Reply:
This makes sense. I guess the only other reason that comes to mind is that this is less error prone regarding typos. I could write `case 'daytine` and not realize it. But this is circumvented as well by writing good tests. I'm gonna change it as you suggested.

Reply:
Actually, I cannot make a typo because TypeScript will catch it.

Reply:
That's one big benefit from the typescript system. If you define a Union then typescript will check it for you :)